### ISSUE TITLE
Vulnerability allow to get access to another user account

### ISSUE THREAD

**Issue by norgalades** [TheHive-Project/TheHive#938]
### Request Type
Bug 

### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | Ubuntu 16.04
| OS version (client)       | Windows 10
| TheHive version / git hash   | 3.2.1-1
| Package Type              | DEB
| Browser type & version    | Internet Explorer Version 11.1689.15063.0

### Problem Description and Steps to Reproduce
Possible vulnerability in TheHive which allows to get access to another user account.

Situation: 
I have created two accounts: user1 and user2 with an equal random password (referred as initial password). 

Steps: 
1.    Successful login with user1 and the initial password: results in logged in as user1
2.    Change of initial password for user1
3.    user1 logs out
4.    Login with user2 (using the same browser) with the initial password -> result: logged in with user1 !!!

The Bug exists using Internet Explorer and only if the browser was not closed between the two logins. It seems that the browser session was not closed correctly.

### Complementary information
Maybe issue 640 can be related even if solved (edge case not considered during the fix).

**Comment by davinerd** [comment]
I think this may help: https://github.com/TheHive-Project/TheHive/issues/864
It's included in 3.3.0

**Comment by nadouani** [comment]
I'll take a look into it, sounds weird :)

**Comment by zpriddy** [comment]
Yeah, this could be that the API calls are getting the cached auth. The API calls may be getting a session cookie that is not expiring correctly, when the call is made again it will auth to being that user. My guess is #864 would fix that issue.

Are there any other open tabs in the browser when you log out of user1?
